CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
   packages.
 * Renamed `Maplibre` to `MapLibre` to be in line with maplibre-native 
   (affects for example the classes MaplibreMap and MaplibreMapController).
+* Renamed iOS plugin classes from `Mapbox` to `MapLibre`.
```

Comment:
I wouldn't consider this change breaking because users won't have to deal with it in any way, or am I wrong?

Reply:
Actually don't really need to list it in the changelog at all :)

Reply:
Moves it to a changes section